exonum/src/sandbox/mod.rs

```diff
@@ -436,11 +435,7 @@ impl Sandbox {
     }
 
     pub fn validators(&self) -> Vec<PublicKey> {
-        self.cfg()
-            .validator_keys
-            .iter()
-            .map(|x| x.consensus_key)
-            .collect()
+        self.cfg().validators.iter().map(|x| x.consensus).collect()
```

Comment:
On one hand `ValidatorKeys` structure obviously contains keys and names of inner fields like `consensus_key` seemed redundant, but however `x.consensus` looks strange.

Reply:
After discussion with @aleksuss we decided that `consensus key` is clearer, even it seems a little bit redundant.